.golangci.yml

```diff
@@ -25,6 +25,7 @@ run:
   #   	vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
   skip-dirs:
     - pkg/binary/envoytest
+    - data/extension/init/templates/tinygo  # skip templated go projects
```

Comment:
I must have syntax wrong.. as the linter (in CI) is still failing like this, presumably because it is peering into these dirs

```
ERRO Running error: context loading failed: failed to load program with go/packages: -: go: downloading github.com/stretchr/testify v1.6.1
```